container/container-core/src/main/java/org/talend/sdk/component/container/ContainerManager.java

```diff
@@ -268,6 +272,10 @@ public String buildAutoIdFromName(final String module) {
             if (autoId.isEmpty()) {
                 throw new IllegalArgumentException("Invalid name for plugin: " + module);
             }
+            final Matcher jiraTicket = versionWithJiraIssue.matcher(autoId);
```

Comment:
Those lines can be after. In case autoId is empty after the matcher check and substring
```
            if (autoId.isEmpty()) {
                throw new IllegalArgumentException("Invalid name for plugin: " + module);
            }
```

Reply:
sure, better place.